src/clj/game/core/runs.clj

```diff
@@ -218,9 +218,10 @@
         cost-as-symbol (when (= 1 (count cost-strs)) (costs-to-symbol cost))
         ;; any trash abilities
         can-steal-this? (can-steal? state side c)
-        trash-ab-cards (->> (concat (all-active state :runner)
-                                    (get-in @state [:runner :play-area]))
-                            (filter #(can-trigger? state :runner (:trash-ability (:interactions (card-def %))) % [c])))
+        trash-ab-cards (when (not= (:zone c) [:discard])
+                         (->> (concat (all-active state :runner)
+                                      (get-in @state [:runner :play-area]))
+                              (filter #(can-trigger? state :runner (:trash-ability (:interactions (card-def %))) % [c]))))
```

Comment:
I'd change the nested keywords to a `get-in`.

Reply:
Heh yeah, I’ll probably go back to `get-in` next time I’m on the computer. I realized I could pull one more level out without thinking about how it’d read later. I personally like how it looks, but only when it’s... conversational? Like when checking prompts in a test. This isn’t quite that, so I think you’re right.